src/quartz_mailer/message.cr

```diff
@@ -0,0 +1,105 @@
+require "email"
+
+class Quartz::Message
+  alias Address = EMail::Address
+
+  property from : Address?
+  property to   = [] of Address
+  property cc   = [] of Address
+  property bcc  = [] of Address
+
+  property subject = ""
+  property text = ""
+  property html = ""
+
+  def self.address(email : String)
+    Address.new email
+  end
+
+  def self.address(name : String, email : String)
+    Address.new email, name
+  end
+
+  def from(email : String) : Nil
+    @from = Message.address email
+  end
+
+  def from(name : String, email : String) : Nil
+    @from = Message.address name, email
+  end
+
+  def from(@from : Address)
+  end
+
+
+  def bcc(email : String) : Nil
+    @bcc << Message.address email
+  end
+
+  def bcc(name : String, email : String) : Nil
+    @bcc << Message.address name, email
+  end
+
+
+
+  def cc(email : String) : Nil
+    @cc << Message.address email
+  end
+
+  def cc(name : String, email : String) : Nil
+    @cc << Message.address name, email
+  end
+
+
+
+  def to(email : String) : Nil
+    @to << Message.address email
+  end
+
+  def to(name : String, email : String) : Nil
+    @to << Message.address name, email
+  end
+
+  def subject(@subject : String) : Nil
+  end
+
+  def text(@text : String) : Nil
+  end
+
+  def html(@html : String) : Nil
+  end
+
+  # for backwards compatibility
+  def body(@text : String) : Nil
+  end
+
+  def to_email
+    EMail::Message.new.tap do |email|
+      email.subject @subject
+
+      if sender = @from
+        email.from sender
```

Comment:
Something is failing here:

```
in spec/quartz_mailer_spec.cr:10: undefined method 'from' for Quartz::Mailer
    mailer.from "test@test.com"
```

Reply:
We need to update README as well 😅

Reply:
@faustinoaq Yeah, I haven't yet done anything with the specs. I wanted to make sure and get some feedback before polishing it completely.